diff -u BIT_STRING.h.orig BIT_STRING.h
--- BIT_STRING.h.orig
+++ BIT_STRING.h
@@ -5,7 +5,7 @@
 #ifndef	_BIT_STRING_H_
 #define	_BIT_STRING_H_
 
-#include <OCTET_STRING.h>	/* Some help from OCTET STRING */
+#include "OCTET_STRING.h"	/* Some help from OCTET STRING */
 
 #ifdef __cplusplus
 extern "C" {
diff -u BOOLEAN.h.orig BOOLEAN.h
--- BOOLEAN.h.orig
+++ BOOLEAN.h
@@ -5,7 +5,7 @@
 #ifndef	_BOOLEAN_H_
 #define	_BOOLEAN_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u IA5String.h.orig IA5String.h
--- IA5String.h.orig
+++ IA5String.h
@@ -5,7 +5,7 @@
 #ifndef	_IA5String_H_
 #define	_IA5String_H_
 
-#include <OCTET_STRING.h>
+#include "OCTET_STRING.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u INTEGER.h.orig INTEGER.h
--- INTEGER.h.orig
+++ INTEGER.h
@@ -5,8 +5,8 @@
 #ifndef	_INTEGER_H_
 #define	_INTEGER_H_
 
-#include <asn_application.h>
-#include <asn_codecs_prim.h>
+#include "asn_application.h"
+#include "asn_codecs_prim.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u NULL.h.orig NULL.h
--- NULL.h.orig
+++ NULL.h
@@ -5,7 +5,7 @@
 #ifndef	ASN_TYPE_NULL_H
 #define	ASN_TYPE_NULL_H
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u NativeEnumerated.h.orig NativeEnumerated.h
--- NativeEnumerated.h.orig
+++ NativeEnumerated.h
@@ -13,7 +13,7 @@
 #ifndef	_NativeEnumerated_H_
 #define	_NativeEnumerated_H_
 
-#include <NativeInteger.h>
+#include "NativeInteger.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u NativeInteger.h.orig NativeInteger.h
--- NativeInteger.h.orig
+++ NativeInteger.h
@@ -12,8 +12,8 @@
 #ifndef	_NativeInteger_H_
 #define	_NativeInteger_H_
 
-#include <asn_application.h>
-#include <INTEGER.h>
+#include "asn_application.h"
+#include "INTEGER.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u NumericString.h.orig NumericString.h
--- NumericString.h.orig
+++ NumericString.h
@@ -5,7 +5,7 @@
 #ifndef	_NumericString_H_
 #define	_NumericString_H_
 
-#include <OCTET_STRING.h>
+#include "OCTET_STRING.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u OCTET_STRING.h.orig OCTET_STRING.h
--- OCTET_STRING.h.orig
+++ OCTET_STRING.h
@@ -5,7 +5,7 @@
 #ifndef	_OCTET_STRING_H_
 #define	_OCTET_STRING_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u asn_SEQUENCE_OF.h.orig asn_SEQUENCE_OF.h
--- asn_SEQUENCE_OF.h.orig
+++ asn_SEQUENCE_OF.h
@@ -5,7 +5,7 @@
 #ifndef	ASN_SEQUENCE_OF_H
 #define	ASN_SEQUENCE_OF_H
 
-#include <asn_SET_OF.h>
+#include "asn_SET_OF.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u asn_codecs_prim.h.orig asn_codecs_prim.h
--- asn_codecs_prim.h.orig
+++ asn_codecs_prim.h
@@ -5,7 +5,7 @@
 #ifndef	ASN_CODECS_PRIM_H
 #define	ASN_CODECS_PRIM_H
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u ber_decoder.h.orig ber_decoder.h
--- ber_decoder.h.orig
+++ ber_decoder.h
@@ -5,7 +5,7 @@
 #ifndef	_BER_DECODER_H_
 #define	_BER_DECODER_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u constr_CHOICE.h.orig constr_CHOICE.h
--- constr_CHOICE.h.orig
+++ constr_CHOICE.h
@@ -6,7 +6,7 @@
 #ifndef	_CONSTR_CHOICE_H_
 #define	_CONSTR_CHOICE_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u constr_SEQUENCE.h.orig constr_SEQUENCE.h
--- constr_SEQUENCE.h.orig
+++ constr_SEQUENCE.h
@@ -5,7 +5,7 @@
 #ifndef	_CONSTR_SEQUENCE_H_
 #define	_CONSTR_SEQUENCE_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u constr_SEQUENCE_OF.h.orig constr_SEQUENCE_OF.h
--- constr_SEQUENCE_OF.h.orig
+++ constr_SEQUENCE_OF.h
@@ -5,8 +5,8 @@
 #ifndef	_CONSTR_SEQUENCE_OF_H_
 #define	_CONSTR_SEQUENCE_OF_H_
 
-#include <asn_application.h>
-#include <constr_SET_OF.h>		/* Implemented using SET OF */
+#include "asn_application.h"
+#include "constr_SET_OF.h"		/* Implemented using SET OF */
 
 #ifdef __cplusplus
 extern "C" {
diff -u constr_SET_OF.h.orig constr_SET_OF.h
--- constr_SET_OF.h.orig
+++ constr_SET_OF.h
@@ -5,7 +5,7 @@
 #ifndef	_CONSTR_SET_OF_H_
 #define	_CONSTR_SET_OF_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u constr_TYPE.h.orig constr_TYPE.h
--- constr_TYPE.h.orig
+++ constr_TYPE.h
@@ -12,8 +12,8 @@
 #ifndef	_CONSTR_TYPE_H_
 #define	_CONSTR_TYPE_H_
 
-#include <ber_tlv_length.h>
-#include <ber_tlv_tag.h>
+#include "ber_tlv_length.h"
+#include "ber_tlv_tag.h"
 
 #ifdef __cplusplus
 extern "C" {
@@ -35,13 +35,13 @@ typedef struct asn_struct_ctx_s {
 	ber_tlv_len_t left;	/* Number of bytes left, -1 for indefinite */
 } asn_struct_ctx_t;
 
-#include <ber_decoder.h>	/* Basic Encoding Rules decoder */
-#include <der_encoder.h>	/* Distinguished Encoding Rules encoder */
-#include <xer_decoder.h>	/* Decoder of XER (XML, text) */
-#include <xer_encoder.h>	/* Encoder into XER (XML, text) */
-#include <per_decoder.h>	/* Packet Encoding Rules decoder */
-#include <per_encoder.h>	/* Packet Encoding Rules encoder */
-#include <constraints.h>	/* Subtype constraints support */
+#include "ber_decoder.h"	/* Basic Encoding Rules decoder */
+#include "der_encoder.h"	/* Distinguished Encoding Rules encoder */
+#include "xer_decoder.h"	/* Decoder of XER (XML, text) */
+#include "xer_encoder.h"	/* Encoder into XER (XML, text) */
+#include "per_decoder.h"	/* Packet Encoding Rules decoder */
+#include "per_encoder.h"	/* Packet Encoding Rules encoder */
+#include "constraints.h"	/* Subtype constraints support */
 
 /*
  * Free the structure according to its specification.
diff -u constraints.h.orig constraints.h
--- constraints.h.orig
+++ constraints.h
@@ -5,7 +5,7 @@
 #ifndef	ASN1_CONSTRAINTS_VALIDATOR_H
 #define	ASN1_CONSTRAINTS_VALIDATOR_H
 
-#include <asn_system.h>		/* Platform-dependent types */
+#include "asn_system.h"		/* Platform-dependent types */
 
 #ifdef __cplusplus
 extern "C" {
diff -u der_encoder.h.orig der_encoder.h
--- der_encoder.h.orig
+++ der_encoder.h
@@ -5,7 +5,7 @@
 #ifndef	_DER_ENCODER_H_
 #define	_DER_ENCODER_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u per_decoder.h.orig per_decoder.h
--- per_decoder.h.orig
+++ per_decoder.h
@@ -5,8 +5,8 @@
 #ifndef	_PER_DECODER_H_
 #define	_PER_DECODER_H_
 
-#include <asn_application.h>
-#include <per_support.h>
+#include "asn_application.h"
+#include "per_support.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u per_encoder.h.orig per_encoder.h
--- per_encoder.h.orig
+++ per_encoder.h
@@ -5,8 +5,8 @@
 #ifndef	_PER_ENCODER_H_
 #define	_PER_ENCODER_H_
 
-#include <asn_application.h>
-#include <per_support.h>
+#include "asn_application.h"
+#include "per_support.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u per_support.h.orig per_support.h
--- per_support.h.orig
+++ per_support.h
@@ -6,7 +6,7 @@
 #ifndef	_PER_SUPPORT_H_
 #define	_PER_SUPPORT_H_
 
-#include <asn_system.h>		/* Platform-specific types */
+#include "asn_system.h"		/* Platform-specific types */
 
 #ifdef __cplusplus
 extern "C" {
diff -u xer_decoder.h.orig xer_decoder.h
--- xer_decoder.h.orig
+++ xer_decoder.h
@@ -5,7 +5,7 @@
 #ifndef	_XER_DECODER_H_
 #define	_XER_DECODER_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u xer_encoder.h.orig xer_encoder.h
--- xer_encoder.h.orig
+++ xer_encoder.h
@@ -5,7 +5,7 @@
 #ifndef	_XER_ENCODER_H_
 #define	_XER_ENCODER_H_
 
-#include <asn_application.h>
+#include "asn_application.h"
 
 #ifdef __cplusplus
 extern "C" {
diff -u xer_support.h.orig xer_support.h
--- xer_support.h.orig
+++ xer_support.h
@@ -6,7 +6,7 @@
 #ifndef	_XER_SUPPORT_H_
 #define	_XER_SUPPORT_H_
 
-#include <asn_system.h>		/* Platform-specific types */
+#include "asn_system.h"		/* Platform-specific types */
 
 #ifdef __cplusplus
 extern "C" {
diff -ur asn_codecs_prim.c.orig asn_codecs_prim.c
--- asn_codecs_prim.c.orig
+++ asn_codecs_prim.c
@@ -2,8 +2,8 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <asn_codecs_prim.h>
+#include "asn_internal.h"
+#include "asn_codecs_prim.h"
 #include <errno.h>
 
 /*
diff -ur asn_SEQUENCE_OF.c.orig asn_SEQUENCE_OF.c
--- asn_SEQUENCE_OF.c.orig
+++ asn_SEQUENCE_OF.c
@@ -2,8 +2,8 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <asn_SEQUENCE_OF.h>
+#include "asn_internal.h"
+#include "asn_SEQUENCE_OF.h"
 
 typedef A_SEQUENCE_OF(void) asn_sequence;
 
diff -ur asn_SET_OF.c.orig asn_SET_OF.c
--- asn_SET_OF.c.orig
+++ asn_SET_OF.c
@@ -2,8 +2,8 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <asn_SET_OF.h>
+#include "asn_internal.h"
+#include "asn_SET_OF.h"
 #include <errno.h>
 
 /*
diff -ur ber_decoder.c.orig ber_decoder.c
--- ber_decoder.c.orig
+++ ber_decoder.c
@@ -2,7 +2,7 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
+#include "asn_internal.h"
 
 #undef	ADVANCE
 #define	ADVANCE(num_bytes)	do {					\
diff -ur ber_tlv_length.c.orig ber_tlv_length.c
--- ber_tlv_length.c.orig
+++ ber_tlv_length.c
@@ -2,9 +2,9 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <ber_tlv_length.h>
-#include <ber_tlv_tag.h>
+#include "asn_internal.h"
+#include "ber_tlv_length.h"
+#include "ber_tlv_tag.h"
 
 ssize_t
 ber_fetch_length(int _is_constructed, const void *bufptr, size_t size,
diff -ur ber_tlv_tag.c.orig ber_tlv_tag.c
--- ber_tlv_tag.c.orig
+++ ber_tlv_tag.c
@@ -2,8 +2,8 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <ber_tlv_tag.h>
+#include "asn_internal.h"
+#include "ber_tlv_tag.h"
 #include <errno.h>
 
 ssize_t
diff -ur BIT_STRING.c.orig BIT_STRING.c
--- BIT_STRING.c.orig
+++ BIT_STRING.c
@@ -2,9 +2,9 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <BIT_STRING.h>
-#include <asn_internal.h>
+#include "asn_internal.h"
+#include "BIT_STRING.h"
+#include "asn_internal.h"
 
 /*
  * BIT STRING basic type description.
diff -ur BOOLEAN.c.orig BOOLEAN.c
--- BOOLEAN.c.orig
+++ BOOLEAN.c
@@ -2,9 +2,9 @@
  * Copyright (c) 2003, 2005 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <asn_codecs_prim.h>
-#include <BOOLEAN.h>
+#include "asn_internal.h"
+#include "asn_codecs_prim.h"
+#include "BOOLEAN.h"
 
 /*
  * BOOLEAN basic type description.
diff -ur constr_CHOICE.c.orig constr_CHOICE.c
--- constr_CHOICE.c.orig
+++ constr_CHOICE.c
@@ -3,9 +3,9 @@
  * All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <constr_CHOICE.h>
-#include <per_opentype.h>
+#include "asn_internal.h"
+#include "constr_CHOICE.h"
+#include "per_opentype.h"
 
 /*
  * Number of bytes left for this structure.
diff -ur constr_SEQUENCE.c.orig constr_SEQUENCE.c
--- constr_SEQUENCE.c.orig
+++ constr_SEQUENCE.c
@@ -3,9 +3,9 @@
  * All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <constr_SEQUENCE.h>
-#include <per_opentype.h>
+#include "asn_internal.h"
+#include "constr_SEQUENCE.h"
+#include "per_opentype.h"
 
 /*
  * Number of bytes left for this structure.
diff -ur constr_SEQUENCE_OF.c.orig constr_SEQUENCE_OF.c
--- constr_SEQUENCE_OF.c.orig
+++ constr_SEQUENCE_OF.c
@@ -3,9 +3,9 @@
  * All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <constr_SEQUENCE_OF.h>
-#include <asn_SEQUENCE_OF.h>
+#include "asn_internal.h"
+#include "constr_SEQUENCE_OF.h"
+#include "asn_SEQUENCE_OF.h"
 
 /*
  * The DER encoder of the SEQUENCE OF type.
diff -ur constr_SET_OF.c.orig constr_SET_OF.c
--- constr_SET_OF.c.orig
+++ constr_SET_OF.c
@@ -3,9 +3,9 @@
  * All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <constr_SET_OF.h>
-#include <asn_SET_OF.h>
+#include "asn_internal.h"
+#include "constr_SET_OF.h"
+#include "asn_SET_OF.h"
 
 /*
  * Number of bytes left for this structure.
diff -ur constr_TYPE.c.orig constr_TYPE.c
--- constr_TYPE.c.orig
+++ constr_TYPE.c
@@ -2,8 +2,8 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <constr_TYPE.h>
+#include "asn_internal.h"
+#include "constr_TYPE.h"
 #include <errno.h>

 /*
diff -ur der_encoder.c.orig der_encoder.c
--- der_encoder.c.orig
+++ der_encoder.c
@@ -2,7 +2,7 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
+#include "asn_internal.h"
 #include <errno.h>
 
 static ssize_t der_write_TL(ber_tlv_tag_t tag, ber_tlv_len_t len,
diff -ur IA5String.c.orig IA5String.c
--- IA5String.c.orig
+++ IA5String.c
@@ -2,8 +2,8 @@
  * Copyright (c) 2003 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <IA5String.h>
+#include "asn_internal.h"
+#include "IA5String.h"
 
 /*
  * IA5String basic type description.
diff -ur INTEGER.c.orig INTEGER.c
--- INTEGER.c.orig
+++ INTEGER.c
@@ -3,9 +3,9 @@
  * All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <INTEGER.h>
-#include <asn_codecs_prim.h>	/* Encoder and decoder of a primitive type */
+#include "asn_internal.h"
+#include "INTEGER.h"
+#include "asn_codecs_prim.h"	/* Encoder and decoder of a primitive type */
 #include <errno.h>
 
 /*
diff -ur NativeEnumerated.c.orig NativeEnumerated.c
--- NativeEnumerated.c.orig
+++ NativeEnumerated.c
@@ -9,8 +9,8 @@
  * implementation deals with the standard (machine-specific) representation
  * of them instead of using the platform-independent buffer.
  */
-#include <asn_internal.h>
-#include <NativeEnumerated.h>
+#include "asn_internal.h"
+#include "NativeEnumerated.h"
 
 /*
  * NativeEnumerated basic type description.
diff -ur NativeInteger.c.orig NativeInteger.c
--- NativeInteger.c.orig
+++ NativeInteger.c
@@ -10,8 +10,8 @@
  * implementation deals with the standard (machine-specific) representation
  * of them instead of using the platform-independent buffer.
  */
-#include <asn_internal.h>
-#include <NativeInteger.h>
+#include "asn_internal.h"
+#include "NativeInteger.h"
 
 /*
  * NativeInteger basic type description.
diff -ur NULL.c.orig NULL.c
--- NULL.c.orig
+++ NULL.c
@@ -2,10 +2,10 @@
  * Copyright (c) 2003, 2005 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <asn_codecs_prim.h>
-#include <NULL.h>
-#include <BOOLEAN.h>	/* Implemented in terms of BOOLEAN type */
+#include "asn_internal.h"
+#include "asn_codecs_prim.h"
+#include "NULL.h"
+#include "BOOLEAN.h"	/* Implemented in terms of BOOLEAN type */
 
 /*
  * NULL basic type description.
diff -ur NumericString.c.orig NumericString.c
--- NumericString.c.orig
+++ NumericString.c
@@ -2,8 +2,8 @@
  * Copyright (c) 2003, 2006 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <NumericString.h>
+#include "asn_internal.h"
+#include "NumericString.h"
 
 /*
  * NumericString basic type description.
diff -ur OCTET_STRING.c.orig OCTET_STRING.c
--- OCTET_STRING.c.orig
+++ OCTET_STRING.c
@@ -3,9 +3,9 @@
  * All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <OCTET_STRING.h>
-#include <BIT_STRING.h>	/* for .bits_unused member */
+#include "asn_internal.h"
+#include "OCTET_STRING.h"
+#include "BIT_STRING.h"	/* for .bits_unused member */
 #include <errno.h>
 
 /*
Only in asn1/patches: quoted_includes2.diff
diff -ur per_decoder.c.orig per_decoder.c
--- per_decoder.c.orig
+++ per_decoder.c
@@ -1,6 +1,6 @@
-#include <asn_application.h>
-#include <asn_internal.h>
-#include <per_decoder.h>
+#include "asn_application.h"
+#include "asn_internal.h"
+#include "per_decoder.h"
 
 /*
  * Decode a "Production of a complete encoding", X.691#10.1.
diff -ur per_encoder.c.orig per_encoder.c
--- per_encoder.c.orig
+++ per_encoder.c
@@ -1,6 +1,6 @@
-#include <asn_application.h>
-#include <asn_internal.h>
-#include <per_encoder.h>
+#include "asn_application.h"
+#include "asn_internal.h"
+#include "per_encoder.h"
 
 static asn_enc_rval_t uper_encode_internal(asn_TYPE_descriptor_t *td, asn_per_constraints_t *, void *sptr, asn_app_consume_bytes_f *cb, void *app_key);
 
diff -ur per_opentype.c.orig per_opentype.c
--- per_opentype.c.orig
+++ per_opentype.c
@@ -2,10 +2,10 @@
  * Copyright (c) 2007 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
-#include <per_support.h>
-#include <constr_TYPE.h>
-#include <per_opentype.h>
+#include "asn_internal.h"
+#include "per_support.h"
+#include "constr_TYPE.h"
+#include "per_opentype.h"
 
 typedef struct uper_ugot_key {
 	asn_per_data_t oldpd;	/* Old per data source */
diff -ur per_support.c.orig per_support.c
--- per_support.c.orig
+++ per_support.c
@@ -3,9 +3,9 @@
  * All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_system.h>
-#include <asn_internal.h>
-#include <per_support.h>
+#include "asn_system.h"
+#include "asn_internal.h"
+#include "per_support.h"
 
 char *
 per_data_string(asn_per_data_t *pd) {
diff -ur xer_decoder.c.orig xer_decoder.c
--- xer_decoder.c.orig
+++ xer_decoder.c
@@ -2,9 +2,9 @@
  * Copyright (c) 2004, 2005 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_application.h>
-#include <asn_internal.h>
-#include <xer_support.h>		/* XER/XML parsing support */
+#include "asn_application.h"
+#include "asn_internal.h"
+#include "xer_support.h"		/* XER/XML parsing support */
 
 
 /*
diff -ur xer_encoder.c.orig xer_encoder.c
--- xer_encoder.c.orig
+++ xer_encoder.c
@@ -2,7 +2,7 @@
  * Copyright (c) 2003, 2004 Lev Walkin <vlm@lionet.info>. All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_internal.h>
+#include "asn_internal.h"
 #include <stdio.h>
 #include <errno.h>
 
diff -ur xer_support.c.orig xer_support.c
--- xer_support.c.orig
+++ xer_support.c
@@ -4,8 +4,8 @@
  * 	All rights reserved.
  * Redistribution and modifications are permitted subject to BSD license.
  */
-#include <asn_system.h>
-#include <xer_support.h>
+#include "asn_system.h"
+#include "xer_support.h"
 
 /* Parser states */
 typedef enum {
